demo/simple/move-first-child.html

```diff
@@ -0,0 +1,50 @@
+<head>
+  <title>Virtual Scoller Demo - Moving children</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1, user-scalable=yes">
+  <style>
+    .scroller {
+        overflow: scroll;
+        height: 400px;
+    }
+  </style>
+</head>
+
+<body>
```

Comment:
I don't care too much but I prefer the minimal style of no head/body tags ([and so does the Google style guide](https://google.github.io/styleguide/htmlcssguide.html#Optional_Tags)). Whereas the html tag is necessary as a place to attach the lang=en.

Reply:
Done.